src/Microsoft.JSInterop/JavaScriptRuntime/src/Microsoft.JSInterop.ts

```diff
@@ -0,0 +1,234 @@
+// This is a single-file self-contained module to avoid the need for a Webpack build
+
+module DotNet {
+  (window as any).DotNet = DotNet; // Ensure reachable from anywhere
```

Comment:
Any reason why this isn't camelCase? just struck me as odd

Reply:
I'm thinking of `DotNet` as being like a static class. It's conventional for class names to be PascalCase in JS, hence static-method-like APIs such as `Promise.race`, `WebAssembly.compile`, etc.